compiler/noirc_frontend/src/hir/comptime/scan.rs

```diff
@@ -91,6 +106,27 @@ impl<'interner> Interpreter<'interner> {
         }
     }
 
+    // Identifiers have no code to execute but we may need to inline any values
+    // of comptime variables into runtime code.
+    fn scan_ident(&mut self, ident: HirIdent, id: ExprId) -> IResult<()> {
+        let definition = self.interner.definition(ident.id);
+
+        match &definition.kind {
+            DefinitionKind::Function(_) => Ok(()),
```

Comment:
Why do not evaluate the ident here as well? Is it because any comptime function would be evaluated separately?